lms/static/sass/discussion/utilities/_variables.scss

```diff
@@ -6,3 +10,8 @@ $forum-color-following: $blue;
 $forum-color-staff: $blue;
 $forum-color-community-ta: $green-d1;
 $forum-color-marked-answer: $green-d1;
+
+// post images
+$post-image-dimension: ($baseline*3);  // image size  + margin
```

Comment:
Sure thing, I can make that change! 

( I originally used "-size", but moved to "-dimension" to echo the learner_profile variables we use for the image-frame there.)